Backbone.Tests.ArchUnit/CleanArchitecture.cs

```diff
@@ -12,6 +12,11 @@ public class CleanArchitecture
             .ResideInAssembly("Backbone.Modules.*", true)
             .As("All Modules");
 
+    private static readonly IObjectProvider<IType> CONSUMER_API_ASSEMBLIES =
+        Types().That()
+            .ResideInAssembly("Backbone.Modules.*.ConsumerApi", true)
+            .As("ConsumerApi Assemblies");
```

Comment:
I would rather use the `MODULES` constant as a base and restrict it to those that end with `ConsumerApi`. This way we don't repeat the `Backbone.Modules` part.